cmd/crybapy/cmd_payer_balance.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+type payerBalanceConfig struct {
+	*payerCommandConfig
+	Account string
+}
+
+func newPayerBalanceCommand(parentConfig *payerCommandConfig) *cobra.Command {
+	config := &payerBalanceConfig{
+		payerCommandConfig: parentConfig,
+	}
+	cmd := &cobra.Command{
+		Use:   "balance ACCOUNT",
+		Short: "Checks the current token balance based with the configured payer implementation",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return checkCmd(doPayerBalance(config, args[0]))
+		},
+	}
+	RegisterFlags(cmd, &config.payerCommandConfig.PayerConfig)
+	return cmd
+}
+
+func doPayerBalance(config *payerBalanceConfig, spenderKeyPath string) error {
+
+	log, err := zap.NewDevelopment()
```

Comment:
shouldn't we use `zap.NewProduction` for the root logger?

Reply:
1959568937eacb0c2f3d1b2a1be068f3e088f131